dgp/lib/trajectory_ingestor.py

```diff
@@ -17,7 +17,7 @@
 
 def import_trajectory(filepath, delim_whitespace=False, interval=0,
                       interp=False, is_utc=False, columns=None, skiprows=None,
-                      timeformat='sow'):
+                      timeformat='sow', engine='python', delim='\s+'):
```

Comment:
I would make the default value comma. I'm pretty sure when we use this elsewhere in the code we assume it's the default delimiter and this would break. Also, use `sep`. `delimiter` is an alternate name for `sep`, but the default value is `None`, so best to keep with `sep` to avoid confusion.